src/lib/views/account-settings/TwoFactAuth.svelte

```diff
@@ -0,0 +1,150 @@
+<script>
+  /* This Source Code Form is subject to the terms of the Mozilla Public
+   * License, v. 2.0. If a copy of the MPL was not distributed with this
+   * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+  import Button from "../../../lib/components/Button.svelte";
+  import "../../../lib/components/auth-cards/Auth.css";
+
+  let password;
+  let passwordEl;
+  let passwordVisible = false;
+  let is2FA = true;
+  let isBlank = true;
+
+  const handleToggle = () => {
+    passwordVisible = !passwordVisible;
+    const type =
+      passwordEl.getAttribute("type") === "password" ? "text" : "password";
+    passwordEl.setAttribute("type", type);
+  };
+
+  const handleAuthToggle = () => {
+    is2FA = !is2FA;
+  };
+
+  const handleChange = () => {
+    console.log("test");
+  };
+</script>
+
+<div class="two-factor-wrapper">
+  <div
+    class="two-factor-content"
+    style="margin-bottom: 24px; box-sizing: content-box;"
+  >
+    <div class="lead-wrapper">
+      <p class="lead" style="padding-top: 20px;">
+        Enable SMS verification to sign in using a one time passcode sent as a
+        SMS to your mobile phone.
+      </p>
+    </div>
+
+    <div class={is2FA ? "factor-btn" : "factor-btn inactive"}>
+      <Button custom="twoFA" on:click={handleAuthToggle} size="lg"
+        >{is2FA ? "Turn on" : "Turn off"}</Button
+      >
+    </div>
+  </div>
+
+  <form method="post">
```

Comment:
Was Ryan's intent to disable the input fields if 2FA is turned off - as you have here - or to hide completely - that's what I see in the figma?

Reply:
I didn't implement the majority of Ryan's designs in this repo. That will be for the upcoming sprint. But yes, the fields will be hidden in the next PR.